cli.js

```diff
@@ -75,7 +79,7 @@ if (cli.flags.init) {
 	return;
 }
 
-var api = new Api(cli.input, {
+var api = new Api(cli.input.length ? cli.input : arrify(conf.files), {
```

Comment:
@kevva Yeah, I think what @jamestalmage meant, was if we have:

package.json
```js
"ava": {
  "files": ["a.js", "b.js"]
}
```

and we want to test just b.js:

```
$ ava b.js
```